schemas/common/telemetry/logger/v1.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "https://mapcolonies.com/common/telemetry/logger/v1",
+  "title": "commonTelemetryLoggerV1",
```

Comment:
no need for common in the title

Reply:
but other examples AKA base and tracing include "common" in the title so I did it unified also in logger

Reply:
done